prompt/prompt-structure/src/commonMain/kotlin/ai/koog/prompt/structure/json/generator/JsonSchemaGenerator.kt

```diff
@@ -78,6 +81,112 @@ public abstract class JsonSchemaGenerator {
 
             return elementDescriptionOverride ?: elementDescriptionAnnotation
         }
+
+
+        /**
+         * Retrieves the maximum allowable numeric value for the descriptor type based on the `@LLMMax`
+         * or `@LLMRange` annotations. If both annotations are present, an exception is thrown.
+         * If neither annotation is found, returns null.
+         *
+         * @return The maximum numeric value defined by the `@LLMMax` or `@LLMRange` annotation,
+         * or null if no maximum value is specified.
+         * @throws IllegalStateException if both `@LLMMax` and `@LLMRange` annotations are present.
+         */
+        public fun getTypeMax(): Int? {
```

Comment:
You don't need `getType` methods for annotations like min/max, since these are applicable to strings/numbers, and you can't really annotate existing library data types. You would use these annotations on class elements only.